html/lesson10.js

```diff
@@ -0,0 +1,195 @@
+/*
+ Bai 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả
+*/
+
+const printTableMultiple = num => {
+
+    //Doi voi truong hop num là undefined hoăc null
```

Comment:
```suggestion
    // Doi voi truong hop num là undefined hoăc null
```
Mọi thứ đều phải dùng tiếng Anh ngoại trừ được yêu cầu dùng ngôn ngữ khác hoặc bắt buộc phải dùng ngôn ngữ khác